src/test/scala/chiselTests/interface/InterfaceSpec.scala

```diff
@@ -301,4 +301,57 @@ class InterfaceSpec extends AnyFunSpec with Matchers {
 
   }
 
+  it("should support ref types that point to subfields of aggregates") {
+    object RefInterface extends Interface {
+
+      final class RefBundle extends Bundle {
+        val r = Output(Probe(Bool()))
+      }
+
+      override type Ports = RefBundle
+
+      override type Properties = Unit
+
+      override val ports = new Ports
+
+    }
+
+    class RefComponent extends RawModule {
+      val w_ref = IO(Output(Probe(Bool())))
+      val w = Wire(new Bundle {
+        val a = UInt(4.W)
+        val b = Bool()
+      })
+      w.a := 2.U(4.W)
+      w.b := true.B
+      val w_probe = ProbeValue(w.b)
+      define(w_ref, w_probe)
+    }
+
+    implicit val refConformance =
+      new ConformsTo[RefInterface.type, RefComponent] {
+        override def genModule() = new RefComponent
+
+        override def portMap = Seq(
+          _.w_ref -> _.r
+        )
+
+        override def properties = {}
+      }
+
+    class RefClient extends RawModule {
+      val x = IO(Output(Bool()))
+      val refInterface = chisel3.Module(new RefInterface.Wrapper.BlackBox)
+      x := read(refInterface.io.r)
+    }
+
+    val dir = new java.io.File("test_run_dir/interface/InterfaceSpec/should-support-ref-types-to-aggregates")
```

Comment:
Can we better derive this directory? There are utility functions to generate test directories.

Reply:
I guess this same feedback applies throughout this file so I won't block this particular PR on it but let's please clean this aspect of the tests up.